### DISCUSSION TITLE
Epic store issue

### DISCUSSION THREAD

**Discussion by Bugboytoad** [Heroic-Games-Launcher/HeroicGamesLauncher#3228] [Q&A]
Hi, new to GitHub, so apologies if this is the wrong place to post this.
Yesterday I claimed a game from Prime via Heroic launcher, but the game was supplied via Epic.  During the claiming process, I was taken to the Epic store to verify linking my prime account to Epic.  Since then, when I navigate to the Epic store in Heroic, it logs me into Prime and I cannot get back to the Epic store.  The game was claimed correctly an I am able to play, just unable to access the Epic store via Heroic.
Does anyone know how to revert the Epic store address back to the correct URL?
Thanks in advance.

**Comment by DDzwiedziu** [comment]
I have the exact same problem, but in reverse. After pairing Epic-Prime (again) I can't get back to Jeff's' "Drink Their Blood" Bezos store.

Edit 2023-12-01: seems that the last/last-last update fixed this problem.
